12-thinking-with-data.Rmd

```diff
@@ -2,10 +2,452 @@
+
+
+
+### Regression modeling {#house-prices-regression}
+
+For now let's focus on the latter, interaction model we've visualized in Figure \@ref(fig:house-price-interaction-2) above. What are the 5 different slopes and intercepts for the condition = 1, condition = 2, ..., and condition = 5 lines in Figure \@ref(fig:house-price-interaction-2)? To determine these, we first need the values from the regression table:
+
+
+```{r}
+# Fit regression model:
+price_interaction <- lm(log10_price ~ log10_size * condition, data = house_prices)
+# Get regression table:
+get_regression_table(price_interaction)
+```
+
+Recall from Section \@ref(model4interactiontable) on how to interpret the outputs where there exists an *interaction term*, where in this case the "baseline for comparison" group for the categorical variable `condition` are the condition 1 houses. We'll write our answers as: 
+
+$$\widehat{\log10(\text{price})} = \hat{\beta}_0 + \hat{\beta}_{\text{size}} * \log10(\text{size})$$
+
+for all five condition levels separately:
+
+1. Condition 1: $\widehat{\log10(\text{price})} = 3.33 + 0.69 * \log10(\text{size})$
+1. Condition 2: $\widehat{\log10(\text{price})} = (3.33 + 0.047) + (0.69 - 0.024) * \log10(\text{size}) = 3.38 + 0.666 * \log10(\text{size})$
+1. Condition 3: $\widehat{\log10(\text{price})} = (3.33 - 0.367) + (0.69 + 0.133) * \log10(\text{size}) = 2.96 + 0.823 * \log10(\text{size})$
+1. Condition 4: $\widehat{\log10(\text{price})} = (3.33 - 0.398) + (0.69 + 0.146) * \log10(\text{size}) = 2.93 + 0.836 * \log10(\text{size})$
+1. Condition 5: $\widehat{\log10(\text{price})} = (3.33 - 0.883) + (0.69 + 0.31) * \log10(\text{size}) = 2.45 + 1 * \log10(\text{size})$
+
+These correspond to the regression lines in the exploratory visualization of the interaction model in Figure \@ref(fig:house-price-interaction-2) below: 
+
+```{r, echo=FALSE, message=FALSE, warning=FALSE, fig.width=16/2, fig.height=9/2}
+ggplot(house_prices, aes(x = log10_size, y = log10_price, col = condition)) +
+  geom_point(alpha = 0.1) +
+  labs(y = "log10 price", x = "log10 size", title = "House prices in Seattle") +
+  geom_smooth(method = "lm", se = FALSE)
+```
+
+For homes of all 5 condition types, as the size of the house increases, the prices increases. This is what most would expect. However, the rate of increase of price with size is fastest for the homes with condition 3, 4, and 5 of 0.823, 0.836, and 1 respectively; these are the 3 most largest slopes out of the 5.
+
+
+### Inference for regression {#house-prices-inference-for-regression}
+
+Contributions via [pull request](http://happygitwithr.com/big-picture.html) to the [ModernDive GitHub repository](https://github.com/moderndive/moderndive_book) are welcome!
```

Comment:
Do we want this included here?

Reply:
I don't have any material I can quickly cut and paste for this section. Let's comment this section out for now, especially since Ch11 isn't done yet.